src/main/java/com/godel/project/javamasteryproject/services/EmployeeConverterToDtoOrEntity.java

```diff
@@ -0,0 +1,49 @@
+package com.godel.project.javamasteryproject.services;
+
+import com.godel.project.javamasteryproject.controllers.dto.EmployeeDto;
+import com.godel.project.javamasteryproject.dao.entities.EmployeeEntity;
+import com.godel.project.javamasteryproject.enums.Gender;
+import com.godel.project.javamasteryproject.services.interfaces.IEmployeeConverterToDtoOrEntity;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.convert.ConversionService;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+@Component
+public class EmployeeConverterToDtoOrEntity implements IEmployeeConverterToDtoOrEntity {
```

Comment:
better name EmployeeConverter

why do you have two class for converter EmployeeConverterToDtoOrEntity and EmployeeRowMapper?